pyatv/protocols/companion/connection.py

```diff
@@ -90,12 +90,18 @@ def enable_encryption(self, output_key: bytes, input_key: bytes) -> None:
         """Enable encryption with the specified keys."""
         self._chacha = chacha20.Chacha20Cipher(output_key, input_key, nonce_length=12)
 
+    def set_listener(self, listener: CompanionConnectionListener) -> None:
```

Comment:
Since `set_listener` should be used from now on, please change the variable name to `_listener` to show this intent. It will require additional change here as well:

https://github.com/postlund/pyatv/blob/f8fbbfe5373cd6554bb5e8d008fbbbe5719bf5c0/pyatv/protocols/companion/protocol.py#L85

Maybe some tests are affected as well, not sure.

Reply:
It's not really my intention to change the API, which is used also in [`MrpAppleTVProxy`](https://github.com/postlund/pyatv/blob/f8fbbfe5373cd6554bb5e8d008fbbbe5719bf5c0/pyatv/scripts/atvproxy.py#L60) and possibly in user code somewhere, just to work around the issue that pylint appears to be incorrectly inferring the type based on ```self.*.listener = self```.

How about [this alternative fix](https://github.com/michalmo/pyatv/commit/b501c91dfb5cb14626bdcaef96a8fbea401452df)? this way the change is localized to `CompanionAppleTVProxy`.

Reply:
This is an internal API, so no one should be using it outside of pyatv. So it's totally fine to change it, I just want to make sure we are consistent and use one way of doing things.

Reply:
Alright, done https://github.com/postlund/pyatv/pull/1912/commits/6be6c7489bc73466929789d2b7e1a2c3646f638c